arangod/RocksDBEngine/RocksDBIndex.h

```diff
@@ -25,6 +25,7 @@
 #define ARANGOD_ROCKSDB_ENGINE_ROCKSDB_INDEX_H 1
 
 #include <rocksdb/status.h>
+#include <rocksdb/write_batch.h>
```

Comment:
I suggest forward-declaring the type here
```suggestion
namespace rocksdb {
class WriteBatch;
}
```